### PR TITLE
Document oGridColumn - fixes #81

### LINKED ISSUE #81: Document oGridColumn mixin in README
Add use cases for `@include oGridColspan` in the README

### PR DESCRIPTION
**Pull request by kaelig**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,18 +106,30 @@ A half width column that becomes full-width on medium screens and up:
...
+```
+
 A column which has `width: auto` on small screens, and then takes half the available space on medium screens and up:
 
 ```html
 <div data-o-grid-colspan="M6"></div>
 ```
 
+```scss
+div { @include oGridColumn((L: two-thirds)); }
```
Comment (AlbertoElias) [inline README.md]: this isn't the same as M6, am I missing something?

File: README.md [hunk 2]
```diff
@@ -257,6 +315,85 @@ Example:
 }
 ```
 
+#### Responsive column helper
+
+For simplicity, examples below don't show in the output code that brings support for Internet Explorer.
```
Comment (AlbertoElias) [inline README.md]: superfluous 'in'

File: README.md (hunk 1 above)
Comment (kaelig) [inline README.md]: Good catch, thanks

**Comment by AlbertoElias** [comment]
lgtm

File: README.md (hunk 2 above)
Comment (kaelig) [inline README.md]: Gracias profesora.

**Pull request merged**